src/components/EncounterPage/EncounterPage.js

```diff
@@ -22,24 +42,92 @@ export default function EncounterPage({ creatureEntries }) {
           />
         ) : (
           <PlayerEntry
+            index={index}
+            activeIndex={activeIndex}
             playerName={creatureEntry.playerName}
             playerArmorClass={creatureEntry.playerArmorClass}
             playerInitiative={creatureEntry.playerInitiative}
             key={creatureEntry._id}
           />
         )
       )}
+      <ButtonContainer>
+        <TurnButton
+          buttonText="Previous Turn"
+          arrowSymbol={'←'}
+          onClick={setLastTurn}
+        />
+        <TurnButton
+          buttonText="Next Turn"
+          arrowSymbol={'→'}
+          onClick={setNextTurn}
+        />
+      </ButtonContainer>
+      <TurnCounterStyled>
+        Turn: {turnNumber}/{creaturesByInitiative.length}
+      </TurnCounterStyled>
+      <RoundCounterStyled>Round: {roundNumber}</RoundCounterStyled>
       <LinkStyled href="/">
         <PageButton onClick={onEndEncounter} buttonText="End Encounter" />
       </LinkStyled>
     </>
   )
 
+  function setNextTurn() {
+    setActiveIndex(
+      activeIndex < creaturesByInitiative.length - 1 ? activeIndex + 1 : 0
+    )
+    setTurnNumber(
+      turnNumber < creaturesByInitiative.length ? turnNumber + 1 : 1
+    )
+    setRoundNumber(
+      turnNumber < creaturesByInitiative.length ? roundNumber : roundNumber + 1
+    )
+  }
+
+  function setLastTurn() {
+    setActiveIndex(
+      activeIndex === 0 && roundNumber === 1
+        ? 0
+        : turnNumber === 1 && roundNumber > 1
+        ? creaturesByInitiative.length - 1
+        : activeIndex - 1
+    )
+    setTurnNumber(
+      turnNumber === 1 && roundNumber === 1
+        ? 1
```

Comment:
Do you really need to set the turn number to 1 if it already is 1? I think you can skip the setTurnNumber in that case. Maybe you can put the ternary operator outside the function argument?